server/etcdserver/api/membership/cluster.go

```diff
@@ -816,6 +812,9 @@ func (c *RaftCluster) SetDowngradeInfo(d *serverversion.DowngradeInfo, shouldApp
 
 // IsMemberExist returns if the member with the given id exists in cluster.
 func (c *RaftCluster) IsMemberExist(id types.ID) bool {
+	// gofail: var sleepAfterIsMemberExist struct{}
+	// defer time.Sleep(time.Second)
+
```

Comment:
You can add a failpoint right above the line 821 "`return ok`",

```
// gofail: var sleepAfterIsMemberExist struct{}
```

and inject `sleep("1s")` to it during test